js/main.js

```diff
@@ -155,6 +164,8 @@ function quizTimer() {
 }
 
 function questionsAndTime() {
+  var $loadingIcon = document.querySelector('.lds-dual-ring');
+  $loadingIcon.remove();
```

Comment:
Instead of creating and destroying this element with every request, I would just hide and show it with a class.